Create ui-c8y-1022-4-11-fixed-displaying-a-link-from-a-device-smart-rules-tab-in.md (#3512)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1022-4-11-fixed-displaying-a-link-from-a-device-smart-rules-tab-in.md

---
date: ""

> title: Readded link from device Smart Rules tab in Cockpit to its corresponding device page in Device Management

Reviewer: ```suggestion
title: Readded link from the Cockpit application to the corresponding device details in the Device Management application
```

> title: Fixed displaying a link from a device Smart Rules tab in Cockpit to the corresponding device page in Device Management (#9441) [GRAFT][release/cd] (#9499)

Reviewer 2: ```suggestion
title: Readded link from device Smart Rules tab in Cockpit to its corresponding device page in Device Management
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YdSEScrEC
    label: Cockpit
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-63963
version: 1022.4.11
---

> This fix restores displaying a link from a device's "Smart Rules" tab in Cockpit to the corresponding device's page in Device Management.

Reviewer 2: ```suggestion
With this fix, it is again possible to navigate from a device's "Smart Rules" tab in Cockpit to its corresponding device page in Device Management by clicking on a link. This restores the expected navigation behavior and allows users to conveniently access device details from Cockpit.
```

> With this fix, it is again possible to navigate from a device's "Smart Rules" tab in Cockpit to its corresponding device page in Device Management by clicking on a link. This restores the expected navigation behavior and allows users to conveniently access device details from Cockpit.

Reviewer: ```suggestion
It is now again possible to navigate via a link on the device's **Smart rules** tab in the Cockpit application to its corresponding device page in the Device Management application. This change restores the expected navigation behavior and allows users to conveniently access device details from the Cockpit application.
```

> It is now again possible to navigate via a link on the device's **Smart rules** tab in the Cockpit application to its corresponding device page in the Device Management application. This change restores the expected navigation behavior and allows users to conveniently access device details from the Cockpit application.

Reviewer: ```suggestion
It is now again possible to navigate via a link on the device's **Smart rules** tab in the Cockpit application to its corresponding device details in the Device Management application. This change restores the expected navigation behavior, allowing users to conveniently access device details from the Cockpit application.
```
